standards/application/segmentation.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Message Segmentation and Reconstruction over Waku
+name: Message Segmentation and Reconstruction
+tags: [waku-application, segmentation]
+version: 0.1
+status: draft
+---
+
+## Abstract
+
+This specification defines an application-layer protocol for **segmentation** and **reconstruction** of messages carried over Waku when the original payload exceeds the maximum Waku's message size. Applications partition the payload into multiple Waku envelopes and reconstruct the original on receipt, even when segments arrive out of order or up to **12.5%** of segments are lost. The protocol uses **Reed–Solomon** erasure coding for fault tolerance. Messages whose payload size is **≤ `segmentSize`** are sent unmodified.
+
+## Motivation
+
+Waku Relay deployments typically propagate envelopes up to **1 MB**. To support larger application payloads (e.g., up to **10 MB** or more), a segmentation layer is required. This specification enables larger messages by partitioning them into multiple envelopes and reconstructing them at the receiver. Erasure-coded parity segments provide resilience against partial loss or reordering.
```

Comment:
The default seems to be 150kb by default in nwaku.
https://github.com/waku-org/nwaku/blob/9808e205af522989cc6888c12e6656df47e0da48/waku/waku_core/message/default_values.nim#L5

You may also want to update the message spec to make it clear and reference it here.